### ISSUE TITLE
find a better way to store API keys

### ISSUE THREAD

**Issue by mikeantonacci** [rectangle-dbmi/PATTrack-Windows#1]
That Resources.resw file is really intended for storing strings for localization purposes, using it this way is kind of a hack.  It's not clear what the correct/standard way to do this is...

**Comment by RitwikGupta** [comment]
So we're supposed to obfuscate and encrypt the key to store it as a string in the app because it can easily be disassembled

**Comment by mikeantonacci** [comment]
Yeah, that's the other (and more important) problem, even when I grepped for the keys in the project folder to make sure I wasn't committing them I saw them in some binary files in the Debug directory...it's an ugly temporary solution so that I could get the keys out of the source code and get something on GitHub faster.

**Comment by RitwikGupta** [comment]
Did you .gitignore those files?

**Comment by mikeantonacci** [comment]
Visual Studio generates a .gitignore by default that includes those build directories and a bunch of other IDE generated nonsense, so they were ignored already.

**Comment by epicstar** [comment]
May not find a better way so if we still see this open and we're ready to release, let's close this issue or move it to another milestone
